tool-testsuite/test/org/antlr/v4/test/tool/TestToolSyntaxErrors.java

```diff
@@ -60,6 +61,16 @@ public void testSetUp() throws Exception {
 		super.testSetUp();
 	}
 
+	@Test
+	public void AllErrorCodesDistinct() {
+		ErrorType[] errorTypes = ErrorType.class.getEnumConstants();
+		for (int i = 0; i < errorTypes.length; i++) {
```

Comment:
maybe add error types to a set then compare length of the set and the number of error types?